provider/metabase/config.go

```diff
@@ -49,9 +49,7 @@ func (c *Credentials) Decrypt(decryptor domain.Decryptor) error {
 	return nil
 }
 
-type PermissionConfig struct {
-	Name string `json:"name" mapstructure:"name" validate:"required"`
-}
+type PermissionConfig string
```

Comment:
```suggestion
type Permission string
```
Does this variable name sound better to you?

Reply:
Yes, LGTM but if we keep PermissionConfig, then it would be consistent with all other providers. WDYT @rahmatrhd ?

Reply:
yes, let's rename all of them then if it is better

Reply:
Sure! Done with the changes.